tests/conftest.py

```diff
@@ -13,6 +17,79 @@
 from lektor.reporter import BufferReporter
 
 
+@pytest.fixture(scope="session", autouse=True)
+def temporary_lektor_cache(tmp_path_factory):
+    """Get Lektor to use a temporary cache directory.
+
+    This prevents the tests from leaving scats behind in the
+    user’s real cache directory.
+
+    """
+    cache_dir = tmp_path_factory.mktemp("lektor_cache")
+
+    # NB: lektor.utils.get_cache_dir() does not make it easy to configure
```

Comment:
I think it's fine that we need to use monkeypatch to change the cache directory. I wouldn't want to add an environment variable that we'd just expect to use internally for tests - if you have other uses for it, that's another story of course. In any case, I think this comment could be shortened a bit, it currently sounds like an open TODO.

Reply:
Okay, I'll just delete that part of the comment.

It seems like it would be polite to provide a general way to configure the location of Lektor’s cache directory, but I don't have a particular use-case in mind.  ... so ... never mind!